plugins/plugin_utils/_hashi_vault_lookup_base.py

```diff
@@ -42,6 +42,9 @@ def parse_kev_term(self, term, plugin_name, first_unqualified=None):
                 else:
                     raise AnsibleError("%s lookup plugin needs key=value pairs, but received %s" % (plugin_name, term))
 
+            if key in param_dict:
+                raise AnsibleOptionsError("Duplicate key '%s' in the term string '%s'" % (key, term))
```

Comment:
I can't make a direct comment/suggestion on a line that didn't change, but you'll probably want to change line 15 to this:

```python
from ansible.errors import AnsibleError, AnsibleOptionsError
```

Reply:
Sure, I've missed this one.